### ISSUE TITLE
Usernames with special characters prevent game from starting

### ISSUE THREAD

**Issue by FaiThiX** [goatcorp/FFXIVQuickLauncher#618]
**Describe the bug here:**
``I cant start game bc of umlauts like "é"``

**Steps to reproduce the bug:**
```
1. Create a Windowsuser like mine "René"
2. Start Launcher and Log in
3. When game starts it crashes with with error
```

**Expected behavior**

Game starts without error 😂 

**Screenshots**

![Errorimg](https://screen.faithix.de/ffxiv_dx11_rvfWVkcCjc.png)

**Other information (*please* complete the following information):**
  Operating System: Windows 11
  XIVLauncher version: 6.1.2.0
  Launching as admin?: Changes Nothing
  Using Steam?: No
  Addons/Plugins enabled: Cleared Pluginfolder

**Comment by Makpptfox** [comment]
Same issue here. the "â" in the user name print the exact same error.

**Comment by goaaats** [comment]
We're aware of this, here's a workaround:

1) Hold shift while starting the launcher, if autologin is enabled.
2) Click the settings gear
3) Click "Dalamud"
4) Switch "Choose how to load Dalamud" to "Legacy"

**Comment by goaaats** [comment]
This should be resolved in version 6.1.3.
